sherpa/astro/io/crates_backend.py

```diff
@@ -314,8 +318,7 @@ def _try_col_list(crate, colname, num, make_copy=False, fix_type=False,
         return numpy.array([None] * num)
 
     col = crate.get_column(colname)
-
-    if hasattr(col, 'is_varlen') and col.is_varlen():
+    if col.is_varlen():
```

Comment:
Do all cols in crates are "is_varlen" and the previous code was just too cautious or why was this needed before, but no longer?

Reply:
Columns are represented by CrateData objects and they all have an is_varlen method. I believe this is just old code from when support for this method was being introduced, but that was a **long** time ago.